### ISSUE TITLE
Newer kernel causes Microsoft X-Box 360 pad emulation to stop working when using DS4 (PS4 controller)

### ISSUE THREAD

**Issue by Whitetyphon** [ValveSoftware/steam-for-linux#6409]
#### Your system information

* Steam client version: july 17 2019 
* Distribution: Manjaro Linux
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
Using the kernel LTS 4.19.59-1-Manjaro trying to play any game (where 360 emu is required) with ds4 in playstation configuration support ends up in games failing to accept input of the device, however when rolling back to an older kernel ( LTS 4.14.133-1-Manjaro) everything works as expected with the emulated xbox 360 input working and the game accepting input

#### Steps for reproducing this issue:

1. Using the Manjaro LTS kernel 4.19-59-1 or newer connect a ds4 controller
2. Enable Playstation configuration support in big picture mode or without, doesn't matter
3. Connect a gaming keyboard that is recognized as a /dev/input/eventX joystick (Corsair k70 was mine)
4. Launch a game
5. Test controller

EDIT:
Seems this issue isn't related to the linux client, but rather to proton, and therefore irrelevant to this issue tracker.

**Comment by Whitetyphon** [comment]
I discovered the reason for everything not working on newer kernels. Having more than two /dev/input/jsx devices (first being ds4, second being emulated 360 pad) connected causes the issue.

When I plugged my keyboard out (because it's recognized as a jsx device on these newer kernels) and just launched a game on the non working kernel everything was working. 

I did some additional testing with a gamepad instead of the keyboard and having it occupy the js2 slot while my ds4 (js0) and emulated xbox 360 pad (js1) occupied the other two. That caused the issue to come back again.

So the issue seems to be that having 3 /dev/input/jsx devices

**Comment by Whitetyphon** [comment]
I have also discovered that inserting the 3rd /dev/input/jsx device after a successful launch of a game with only 2 /dev/input/jsx devices connected, everything still works as intended.

So it seems that having 3 /jsx devices connected is only hindering the _creation_ or maybe hooking of the emulated 360 controller while the game is booting up, but when the emulated controller is already created/or hooked into you can connect as many /dev/input/jsx devices you want

**In short**
You can reconnect every /dev/input/jsx device you want after launching with the 2 /jsx device workaround, and have everything working (Not ideal, because I have to re plug keyboard every time
